src/ItemEdit.js

```diff
@@ -62,68 +97,222 @@ const ItemEdit = ({ index, roadmap, onChange, onDone }) => {
-                direction="row-responsive"
-                align="start"
-                justify="between"
-                gap="small"
-                margin={{ bottom: 'medium' }}
-              >
-                <FormField name="section" htmlFor="section" margin="none">
-                  <TextInput
-                    name="section"
-                    id="section"
-                    placeholder="Section"
-                    suggestions={roadmap.sections}
-                  />
-                </FormField>
-                <FormField
-                  name="target"
-                  htmlFor="target"
-                  required
-                  margin="none"
-                >
-                  <DateInput name="target" id="target" format="mm/dd/yyyy" />
-                </FormField>
-                <FormField name="label" htmlFor="label" margin="none">
-                  <TextInput
-                    name="label"
-                    id="label"
-                    placeholder="Label"
-                    suggestions={roadmap.labels.map(({ name }) => name)}
-                  />
-                </FormField>
-              </Box>
-              <FormField name="name" htmlFor="name" required>
+              <FormField name="section" htmlFor="section" margin="none">
                 <TextInput
-                  name="name"
-                  id="name"
-                  size="large"
-                  placeholder="Label"
+                  icon={<FormDown />}
+                  reverse
+                  name="section"
+                  id="section"
+                  placeholder="Template"
+                  suggestions={roadmap.sections}
                 />
               </FormField>
+              <FormField name="name" htmlFor="name" required>
+                <TextInput name="name" id="name" placeholder="Title" />
+              </FormField>
               <FormField
                 name="note"
                 htmlFor="note"
                 margin={{ bottom: 'medium' }}
               >
-                <TextArea name="note" id="note" placeholder="Note" />
+                <TextArea name="note" id="note" placeholder="Notes" />
               </FormField>
+              {linkFields.map((linkField, index) => (
+                <Box direction="row" key={`${linkField}~${index}`}>
+                  <FormField
+                    fill="horizontal"
+                    name={`${index}linkUrl`}
+                    htmlFor={`${index}linkUrl`}
+                  >
+                    <TextInput
+                      onChange={(event) => handleLinkInputChange(index, event)}
+                      name={`${index}linkUrl`}
+                      id={`${index}linkUrl`}
+                      value={linkFields[`${index}`].linkUrl}
+                      placeholder="URL"
+                    />
+                  </FormField>
+                  <Box
+                    align="end"
+                    width="40px"
```

Comment:
Can we be more standard with the width and round?

Reply:
Yes, I will switch these to grommet t-shirt sizes